File: IMERA-JAVA/ImeraServidorRMI/ImeraServidorRMI/src/pe/edu/pucp/imera/interfaces/CambioDeEstadoBO.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Interface.java to edit this template
 */
package pe.edu.pucp.imera.interfaces;

import java.rmi.Remote;
import java.rmi.RemoteException;
import pe.edu.pucp.imera.horario.model.EstadoAula;

/**
 *
 * @author cesar
 */
public interface CambioDeEstadoBO extends Remote {
    
    Integer insertar(String idPabellon, Integer idPiso, Integer idAula, Integer idHora, EstadoAula estadoInicial, EstadoAula estadoFinal, Integer idAuxiliar) throws RemoteException;
    
}

File: IMERA-CS/ImeraWA/ImeraUsuarioBO/CambioDeEstadoBO.cs
using ImeraBaseBO;
using ImeraBaseBO.ImeraServicio;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ImeraUsuarioBO
{
    public class CambioDeEstadoBO:BaseBO
    {
        public CambioDeEstadoBO() : base()
        {

        }
        public int insertar(string idPabellon,int idPiso, int idAula,int idHora,estadoAula estadoInicial,
            estadoAula estadoFinal,int idAuxiliar)
        {
            return WsCliente.cambioEstado_insertar(idPabellon,idPiso,idAula,idHora,estadoInicial,estadoFinal, idAuxiliar);
        }
    }
}
